ios/Framework/Themes/Default/Portlet/cordova/ScreensBridgePlugin.m

```diff
@@ -20,7 +20,7 @@ @implementation ScreensBridgePlugin
 
 - (void)postMessage:(CDVInvokedUrlCommand*)command {
 
-	CDVPluginResult* pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:@""];
+	/*CDVPluginResult* pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:@""];
 
```

Comment:
Cordova won't work if this is commented

Reply:
sorry, this was commented to do some tests :)